ouroboros-consensus-cardano-tools/test/disk/config/config.json

```diff
@@ -8,6 +8,7 @@
   "ApplicationName": "cardano-sl",
   "ApplicationVersion": 0,
   "ByronGenesisFile": "byron/genesis.json",
+  "ConwayGenesisFile": "genesis-conway.json",
```

Comment:
TODO another piece of the theme: does the re-scoped Conway still need a genesis file?